modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -177,3 +210,80 @@ function inope_policy_review_form_submit($form, &$form_state) {
   // Rebuild the form.
   $form_state['rebuild'] = TRUE;
 }
+
+/**
+ * Select query for the current node policy review notes.
+ * 
+ * Query the policy review table for any records related to the current
+ * node, note status, and are from the current user.
+ */
+function inope_policy_review_select_query($node, $account, $status) {
```

Comment:
By using status here - It sort of then says -- that only if a user has submitted that they approve the policy will we hide the form. If they have answered 1 and provided text then -- it will return results and then we will build the form -- which isn't what we want in either this challenge or challenge 3.

So from the above you pass in 0 as a status as a hard coded value meaning that - if they selected 1 as their initial response - or that they do have comments this will return results and rebuild the form.

What we actually need to evaluate is -- did this user submit anything since the most recent timestamp on the most recent vid.  If they have a timestamp that is >= to that timestamp then they don't need to see the form.